pkgs/os-specific/darwin/discrete-scroll/default.nix

```
@@ -22,15 +21,15 @@ stdenv.mkDerivation rec {
   '';
 
   installPhase = ''
-    mkdir -p $out/bin
-    cp ./dc $out/bin/discretescroll
+    mkdir -p "$out/bin"
+    cp ./dc "$out/bin/discretescroll"
   '';
 
-  meta = with lib; {
+  meta = with stdenv.lib; {
```

```suggestion
  meta = with stdenv.lib; {
```

This is deprecated by recent contributing change.